### DISCUSSION TITLE
Stop detection for specific object

### DISCUSSION THREAD

**Discussion by neomancernz** [blakeblackshear/frigate#15446] [Ask A Question]
Hi all

Recently started using Frigate, and I am loving it so far. I am having one issue however. I have a covered workbench in my yard, and it keeps detecting it as a person. I have believe I have set an object exception. but it is still showing up. I am not sure if I have done something wrong or if there is an issue with my setup. I have added pictures of my object mask setup and the detections from last night.

Any help appreciated.
![Screenshot 2024-11-28 203612](https://github.com/user-attachments/assets/79d6ced2-f5d8-4ba0-8e09-73ca5f4aa2bd)
![Screenshot 2024-11-28 203528](https://github.com/user-attachments/assets/dcd90315-022c-4ec1-a8f6-5984074d28eb)
![Screenshot 2024-12-11 090850](https://github.com/user-attachments/assets/5cc8d1c7-9aad-43bc-a1ef-ccdbfdada962)

**Comment by NickM-27** [comment]
I would suggest reading the docs on object masks, yours is not being used correctly https://docs.frigate.video/configuration/masks#object-filter-masks

You need to make sure the bottom of the box is included in the object mask
